### PR TITLE
Add AirMount Latest

### PR DESCRIPTION
**Pull request by elliotthilaire**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by elliotthilaire** [comment]
As it is `brew cask uninstall ...` will uninstall everything that has been installed with `brew cask install ...`

Perhaps it is better not to remove 'com.github.osxfuse.pkg.Core' in case it has been installed independently, and instead issue a warning?

File: Casks/airmount.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  pkg 'AirMount.pkg'
+
+  uninstall quit:    'com.tinkerstuff.AirMount',
+            pkgutil: [
+                       'com.tinkerstuff.AirMount',
+                       'com.github.osxfuse.pkg.Core',
+                     ],
+            delete:  '/Applications/AirMount.app'
+
+  caveats 'As part of the AirMount installation, a library called Fuse for OSX (OSXFUSE) will be installed if not already installed on your Mac. For more information on OSXFUSE, check out osxfuse.github.io/'
```
Comment (reitermarkus) [inline Casks/airmount.rb]: Better add `depends_on cask: 'osxfuse'` to the cask to make sure this is installed.

Comment (elliotthilaire) [inline Casks/airmount.rb]: @reitermarkus AirMount comes with, and will install the core part of osxfuse that it needs, so `depends_on cask: 'osxfuse'` is not strictly necessary. 

`depends_on cask: 'osxfuse'` will also install a few extra things such a preference pane, that are not needed for airmount.

I will re-add it as it makes it a bit more clear to manage osxfuse and airmail via cask.

File: Casks/airmount.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  depends_on cask: 'osxfuse'
+
+  pkg 'AirMount.pkg'
+
+  uninstall quit:    'com.tinkerstuff.AirMount',
+            pkgutil: 'com.tinkerstuff.AirMount',
+            delete:  '/Applications/AirMount.app'
+
+  caveats 'As part of the AirMount installation, a library called Fuse for OSX (OSXFUSE) will be installed if not already installed on your Mac. For more information on OSXFUSE, check out osxfuse.github.io/'
```
Comment (jawshooah) [inline Casks/airmount.rb]: Now that you've added the `osxfuse` dependency, this caveat isn't necessary.

Comment (elliotthilaire) [inline Casks/airmount.rb]: Thanks @jawshooah. I've removed the caveat.

**Pull request merged**